cloudify_rest_client/execution_schedules.py

```diff
@@ -0,0 +1,258 @@
+from cloudify_rest_client.responses import ListResponse
+
+
+class ExecutionSchedule(dict):
+    """
+    Cloudify snapshot.
+    """
+
+    def __init__(self, execution_schedule):
+        super(ExecutionSchedule, self).__init__()
+        self.update(execution_schedule)
+
+    @property
+    def id(self):
+        """
+        :return: The name of the execution schedule.
+        """
+        return self.get('id')
+
+    @property
+    def deployment_id(self):
+        """
+        :return: The deployment's id which the scheduled execution is
+        related to.
+        """
+        return self.get('deployment_id')
+
+    @property
+    def workflow_id(self):
+        """
+        :return: The id of the workflow which the scheduled execution runs.
+        """
+        return self.get('workflow_id')
+
+    @property
+    def parameters(self):
+        """
+        :return: The scheduled execution's parameters
+        """
+        return self.get('parameters') or {}
+
+    @property
+    def created_at(self):
+        """
+        :return: Timestamp of the execution schedule creation.
+        """
+        return self.get('created_at')
+
+    @property
+    def next_occurrence(self):
+        """
+        :return: The calculated next date and time in which the scheduled
+        workflow should be executed at.
+        """
+        return self.get('next_occurrence')
+
+    @property
+    def since(self):
+        """
+        :return: The earliest date and time the scheduled workflow should be
+        executed at.
+        """
+        return self.get('since')
+
+    @property
+    def until(self):
+        """
+        :return: The latest date and time the scheduled workflow may be
+        executed at (if any).
+        """
+        return self.get('until')
+
+    @property
+    def stop_on_fail(self):
+        """
+        :return: Whether the scheduler should stop attempting to run the
+        execution once it failed.
+        """
+        return self.get('stop_on_fail')
+
+
+class ExecutionSchedulesClient(object):
+
+    def __init__(self, api):
+        self.api = api
+        self._uri_prefix = 'execution-schedules'
+
+    def create(self, schedule_id, deployment_id, workflow_id, parameters=None,
```

Comment:
i would suggest grouping the parameters, and having the create method accept a "execution_arguments" or something dict, so that there's only 50 arguments to this function and not ten thousand